src/app/RecordingList/RecordingsDataTable.tsx

```diff
@@ -36,33 +36,55 @@
  * SOFTWARE.
  */
 import * as React from 'react';
-import { DataList, DataListCell, DataListCheck, DataListItem, DataListItemCells, DataListItemRow } from '@patternfly/react-core';
+import { Bullseye, DataList, DataListCell, DataListCheck, DataListItem, DataListItemCells, DataListItemRow, Spinner, Text } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
 
 export interface RecordingsDataTableProps {
   toolbar: React.ReactElement;
   tableColumns: string[];
   listTitle: string;
   isHeaderChecked: boolean;
+  isLoading: boolean;
+  isConnectionError: boolean;
   onHeaderCheck: (checked: boolean) => void;
 }
 
 export const RecordingsDataTable: React.FunctionComponent<RecordingsDataTableProps> = (props) => {
-  return (<>
-    { props.toolbar }
-    <DataList aria-label={props.listTitle}>
-      <DataListItem aria-labelledby="table-header-1">
-        <DataListItemRow>
-          <DataListCheck aria-labelledby="table-header-1" name="header-check" onChange={props.onHeaderCheck} isChecked={props.isHeaderChecked} />
-          <DataListItemCells
-            dataListCells={props.tableColumns.map((key , idx) => (
-              <DataListCell key={key}>
-                <span id={`table-header-${idx}`}>{key}</span>
-              </DataListCell>
-            ))}
-          />
-        </DataListItemRow>
-      </DataListItem>
-      { props.children }
-    </DataList>
-  </>);
+  if (props.isConnectionError) {
+    return (<>
+      <br/>
+      <Bullseye>
+        <ExclamationCircleIcon size='md' color='Red'/>
+      </Bullseye>
+      <Bullseye>
+        <Text>
+          Unable to connect
+        </Text>
+      </Bullseye>
+    </>)
+  } else {
+    return props.isLoading? (<>
```

Comment:
Since we already have this whole thing wrapped in an outer `if/else`, could you refactor this to remove the ternary statement?

ie we have

```
if (A) {
  return (<TemplateA/>);
} else {
  return B ? (<TemplateB/>) : (<TemplateC/>);
}
```

This seems like it would be easier to read like:

```
if (A) {
  return (<TemplateA/>);
} else if (B) {
  return (<TemplateB/>);
} else {
 return (<TemplateC/>);
}
```